.github/workflows/unit-tests.yaml

```diff
@@ -28,6 +28,10 @@ jobs:
           distribution: 'corretto'
           java-version: '21'
 
+      - uses: arduino/setup-protoc@v3
```

Comment:
adruino...?

Reply:
Yeah I was surprised too. But it was the most popular package providing `protoc` :sweat_smile: